app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,18 @@
+class UserMailer < ApplicationMailer
+  default from: Rails.configuration.address_for_transactional_authentication_emails
```

Comment:
I was hoping to only use this email address for Devise stuff, but maybe it makes sense here? Might be good to 🪑  . Not strictly a blocker.

Reply:
I need things to come from a no-reply email, and thats what address_for_transactional_authentication_emails does in demo + prod. Seems like we should have an address_for_transactional_emails and not worry about having a separate email address for "authentication" emails -- maybe I'm missing some context on why we'd want something seperate just for devise emails, it doesn't seem necessary when we're ultimately sending from no-reply in demo and prod.